pkg/services/ngalert/models/alert_rule.go

```diff
@@ -344,7 +344,10 @@ type ListAlertRulesQuery struct {
 }
 
 type GetAlertRulesForSchedulingQuery struct {
-	Result []*AlertRule
+	PopulateFolders bool
+
+	Rules         []*AlertRule
+	FoldersTitles map[string]string
```

Comment:
I am concerned that it's not obvious this is a query result rather than a query constraint?